website/docs/community.md

```diff
@@ -28,4 +28,4 @@ This page lists community applications or tools that can be used alongside Komga
 
 ## MangaBox
 
-> [MangaBox](https://github.com/zpaolo11x/mangabox) is a simple Electron client for Komga.
\ No newline at end of file
+> [MangaBox](https://zpaolo11x.github.io/mangabox/) is a streamlined multi-platform client for Komga.
```

Comment:
we can remove the entry entirely in this file, as the Community page aims to have a small paragraph for projects that are not covered elsewhere in the documentation.